수정 완료

jiunsu opened the pull request.

jiunsu committed:
jiunsu test commit

jiunsu committed:
jiunsu fix

sjp38 commented:
무슨 수정인지 더 자세히 설명해 주시면 좋을 것 같아요

ganadist commented:
리팩토링 하신 것 같은데요.
굳이 원래 코드를 남기지 않아도 됩니다.
로그에 변경사항을 정확하게 기재해주세요.

그리고 최신 소스와 sync해주시기 바랍니다.

jiunsu committed:
jiunsu fix2

jiunsu commented:
숫자 입력은 따로 받지 않고 임의로 코드에 설정해놓았습니다. (0~46까지)

jiunsu committed:
jiunsu 수정내용

1 fib 함수를 메인에서 호출하도록함.
2 fib함수 내용: 숫자가 2 보다 작을 경우 자신을 return 하고, 아닌 경우 재귀함수로 바로 전, 바로 이전 숫자를 더하여 계산하도록 함.

jiunsu commented:
다시 로그에 변경내용 상세히 넣어서 commit & push 했습니다.
